hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/HederaWorldState.java

```diff
@@ -230,6 +265,16 @@ public long getSbhRefund() {
             return sbhRefund;
         }
 
+        @Override
+        public Address priorityAddress(Address addressOrAlias) {
+            return null;
+        }
+
+        @Override
+        public Address newAliasedContractAddress(Address sponsor, Address alias) {
+            return null;
+        }
```

Comment:
These implementations seem strange. If we do not require these methods to be implemented in `hedera-services` at this hierarchy level, it probably means we need to move them in another interface or another suitable place. It does not seem right to change or add empty methods inside `hedera-services` just to serve the needs for `mirror-node`.